cmd/client-fs.go

```diff
@@ -1194,3 +1194,11 @@ func (f *fsClient) DeleteEncryption(ctx context.Context) *probe.Error {
 		APIType: "filesystem",
 	})
 }
+
+// Gets bucket info
+func (f *fsClient) GetInfo(ctx context.Context) (BucketInfo, *probe.Error) {
```

Comment:
GetBucketInfo() as a function name looks better.

Reply:
Or generalize it GetStatInfo() which returns StatInfo